### DISCUSSION TITLE
Completely skip `PlayerLoader` screen on retry by pressing specific keybind

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#27480] [Ideas] [answered]
I guess most of the stable players want [this](https://streamable.com/eks8s0) feature back. So as compromise I propose implementing it via keybind only, adding something like "Instant retry (hold)"

![513456t2345yt24y](https://github.com/ppy/osu/assets/86934170/34f3af79-199d-406f-8718-0d43af313358)

**Comment by peppy** [comment]
If anything about the retry was changed, it would affect all flows. The loading screen is part of the flow.

**Reply by Loreos7** [reply]
Previously [related PR](https://github.com/ppy/osu/pull/19594) with reduced loading time was rejected but loading time shown there was good enough to play with. Maybe this could be done by hiding PlayerLoader screen, load as fast as possible so it would be kinda instant?

**Comment by Loreos7** [comment, marked as answer]
https://github.com/ppy/osu/pull/30603
